desktop/cmp/grid/impl/ColChooserModel.js

```diff
@@ -26,11 +27,26 @@ export class ColChooserModel {
     // Show in popover
     @observable isPopoverOpen = false;
 
+    commitOnChange;
+    showRestoreDefaults;
+
     /**
-     * @param {GridModel} gridModel - model for the grid to be managed.
+     * @param {Object} c - ColChooserModel config
+     * @param {GridModel} c.gridModel - model for the grid to be managed.
+     * @param {boolean} [c.commitOnChange] - Immediately render changed columns on grid.
+     *      Set to false to show Save button to commit changes on save. Default true.
+     * @param {boolean} [c.showRestoreDefaults] - show Restore Defaults button. Default true.
+     * @param {number} [c.width] - chooser width for popover and dialog
+     * @param {number} [c.height] - chooser height for popover and dialog
      */
-    constructor(gridModel) {
+    constructor({gridModel, commitOnChange, showRestoreDefaults, width, height}) {
```

Comment:
easiest and also most self documenting to default all four of these in the method signature